specifyweb/specify/datamodel.py

```diff
@@ -6192,7 +6192,7 @@
         classname='edu.ku.brc.specify.datamodel.SpAuditLog',
         table='spauditlog',
         tableId=530,
-        system=True,
+        system=False,
```

Comment:
do we also need to remove it from here?
https://github.com/specify/specify7/blob/69391723254d4ea71df4701a869c225dc00b80dd/specifyweb/specify/load_datamodel.py#L484C6-L484C12

if so, why are there two places designating what tables are system? that is a recipe for bugs

Reply:
Yes, it can (and should) be removed from there.
Although thankfully, it wouldn't have caused any bugs. 

Prior to #4764, that was used to mark which tables are systems when the datamodel was autogenerated from the `specify_datamodel.xml` file from Specify 6. 
Now, it is only used when the datamodel code is regenerated (which should now never be done without some refactoring and a note of which changes we have since made to the `datamodel.py` file). 

I had hoped that the implementation of #4764 would allow the Models and/or Datamodel to be autogenerated instead of manually edited (much like how the frontend Datamodel is generated), but that is not the case currently.